editor/src/document/movement_handler.rs

```diff
@@ -321,7 +334,6 @@ impl MessageHandler<MovementMessage, (&Document, &InputPreprocessor)> for Moveme
 			IncreaseCanvasZoom,
 			DecreaseCanvasZoom,
 			WheelCanvasTranslate,
-			ZoomCanvasToFitAll,
```

Comment:
Should this be removed, or just renamed to its new name of `FitViewportToBounds`?

Reply:
Yes since the document now gets the keyboard shortcut and sends a message to the movement handler